packages/closer/pages/stay/index.tsx

```
@@ -116,19 +116,25 @@ const StayPage = ({
-              t.raw('stay_title')
-            }} />
-                        
+          <Heading
+            level={1}
+            className={`${
+              APP_NAME === 'lios' ? 'text-xl sm:text-2xl' : 'text-4xl'
+            }  pb-2 mt-8`}
+          >
+            <p
+              className="font-accent"
+              dangerouslySetInnerHTML={{ __html: t.raw('stay_title') }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)